assembl/static2/js/app/components/login/sendPwdForm.jsx

```diff
@@ -2,27 +2,64 @@ import React from 'react';
 import { connect } from 'react-redux';
 import { Translate, I18n } from 'react-redux-i18n';
 import { form, FormGroup, FormControl, Button } from 'react-bootstrap';
+import inputHandler from '../../utils/inputHandler';
+import { requestPasswordChangeAction } from '../../actions/authenticationActions';
 
 class SendPwdForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { identifier: null };
+    this.submitHandler = this.submitHandler.bind(this);
+    this.handleInput = this.handleInput.bind(this);
+    this.messageHandler = this.messageHandler.bind(this);
+  }
+
+  handleInput(e) {
+    inputHandler(this, e);
+    const el = document.getElementById('sendPwdForm-error');
+    if (el) { el.innerHTML = ''; }  // BUG: Only shows once; if fails beyond first time, no error shown!
```

Comment:
Can you use `dangerouslySetInnerHTML` instead of `innerHTML `
https://facebook.github.io/react/docs/dom-elements.html

We should fix the bug instead of commit `//BUG...`